.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
I am working on rust-vmm-ci. While writing the README.md I also wanted to give an example of a custom hook that can be placed in the repository. I took linux-loader as an example. You can use the code there to download the bzimage:
https://github.com/andreeaflorescu/rust-vmm-ci/tree/first_buildkite_pipeline#custom-repository-hooks

Some notes:
- We shouldn't rely on the name of the deb being downloaded. In my script I download the deb, extract it and copy the bzimage file to `linux-loader/src` with the name `bzimage`. This means that in the unit tests you will write bzimage instead of `./boot/vmlinuz-4.9.0-9-amd64`:

```rust
    fn make_bzImage() -> Vec<u8> {
        let mut v = Vec::new();
        v.extend_from_slice(include_bytes!("bzimage"));
        v
    }
```
- We should also take care of the clean-up; delete the downloaded files and only keep the bzimage you're actually using
- linux-4.9 is kinda ancient. should we use a newer image?

Reply:
@andreeaflorescu Good person! :-) yes, I will see to use a new version.

Reply:
@sameo  it seems in the container which docker started, it could not touch github, network connection failed, I guess it's caused by the lack of network proxy which we used in host. That's to say we currently can not do the test local.